packages/edit-site/src/components/add-new-template/index.js

```diff
@@ -1,31 +1,431 @@
+/**
```

Comment:
I've moved the old new-template file into this index file. There's no need to separate these two things and there's no need for props since this component is now used only for templates and only in the data views page.